README.md

```diff
@@ -1 +1 @@
-# Adafruit_IS31FL3731 [![Build Status](https://travis-ci.com/adafruit/Adafruit_IS31FL3731.svg?branch=master)](https://travis-ci.com/adafruit/Adafruit_IS31FL3731)
\ No newline at end of file
+# Adafruit_IS31FL3731 [![Build Status](https://github.com/adafruit/Adafruit_IS31FL3731/workflows/Arduino%20Library%20CI/badge.svg)](https://github.com/adafruit/Adafruit_IS31FL3731/actions)
```

Comment:
Please also update this readme using the updated layout, and do the safe for future libraries that you migrate

see for an example:
https://github.com/adafruit/Adafruit_ZeroPDM/tree/actionci

Reply:
Will do